pkgs/development/libraries/qt-5/5.5/qtbase/setup-hook.sh

```diff
@@ -91,7 +91,7 @@ _qtSetQmakePath() {
 
 if [ -z "$NIX_QT5_TMP" ]; then
     if [ -z "$NIX_QT_SUBMODULE" ]; then
-        NIX_QT5_TMP=$(mktemp -d)
+        NIX_QT5_TMP=$(pwd)/__nix_qt5__
```

Comment:
> It will make, because /tmp path was derandomized year ago. I have dedicated /var/buildroot filesystem, and paths inside sandbox still looks like /tmp/${pkg}-drv-0/

This is only half true: if you use the `--keep-failed` option, then the old build directory will be kept.  If you then build the derivation again the zero gets incremented and you are now in `/tmp/${pkg}-drv-1`, then in `/tmp/${pkg}-drv-2`, and so on.